datasetinsights/io/downloader/unity_simulation.py

```diff
@@ -0,0 +1,116 @@
+import logging
+import os
+import re
+from pathlib import Path
+
+from datasetinsights.io.downloader.base import DatasetDownloader
+from datasetinsights.io.usim import Downloader, download_manifest
+
+logger = logging.getLogger(__name__)
+
+
+class UnitySimulationDownloader(DatasetDownloader):
+    """ Downloads unity simulation datasets
```

Comment:
This docstring needs to be improved. First, you should describe `Unity Simulation` or at least provide links to official documentation. Second, we should document expected source uri as well as different override behavior here. Third, all `Args` and `Attributes` should be documented.